.github/workflows/docs.yml

```diff
@@ -20,7 +20,8 @@ jobs:
           python3 -m pip install setuptools
           python3 -m pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
           python3 -m pip install matplotlib
-          python3 -m pip install --user "git+https://github.com/pytorch/data.git"
+          python3 -m pip install -r https://raw.githubusercontent.com/pytorch/data/main/requirements.txt
+          python3 -m pip install --no-build-isolation git+https://github.com/pytorch/data
```

Comment:
You can add a separate step to compile torchdata from local source as we have a prior step which has already checked out torchdata repo.
```
python3 -m pip install -r requirements.txt
python3 setup.py install
```